eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -175,3 +179,20 @@ def _force_create_enrollment(username, course_id, mode, is_active):
     except Exception as err:  # pylint: disable=broad-except
         raise APIException(repr(err))
     return enrollment
+
+
+def _validate_org(course_id):
+    """
+    Validates the course organization against all possible orgs for the site
+    """
+    course_key = CourseKey.from_string(course_id)
+    current_site_orgs = get_current_site_orgs() or []
+    all_orgs = get_all_orgs()
+    orgs_to_exclude = [x for x in all_orgs if x not in current_site_orgs]
+
+    if not current_site_orgs:
+        if course_key.org in orgs_to_exclude:
```

Comment:
since you can only reach this in the event `current_site_orgs` is empty, then line 191 is not doing much. You can directly check `all_orgs` and remove the #191

Reply:
ready @felipemontoya